docs/developers/how-to-refine-plugins.md

```diff
@@ -85,7 +84,7 @@ We use `zod` to validate data. Here's an example from our codebase:
 /**
  * Checks for required fields in input.
  */
-private validateInput(input: ModelParams) {
+private validateInput(input: pluginParams) {
```

Comment:
change function declaration to this one 
`const validateInput = (input: PluginParams) => {`